mantidproject/mantid
# Add Run Summation Tab to SANS v2 GUI

This PR adds a 'Sum Runs' page to the SANS v2 Experimental interface, with functionality similar to that of the 'Add Runs' tab in the old SANS GUI.

**To test:**
1. Open the SANS v2 Experimental Interface `Intefaces->SANS->ISIS SANS v2 experimental` and open the Sum Runs Page and ensure that it meets scientific requirements.
2. Code Review

Fixes #21096 and fixes #21097. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
